akka-projection-core/src/main/resources/reference.conf

```diff
@@ -14,18 +18,24 @@ akka.projection {
     save-offset-after-duration = 500 ms
   }
 
+  # The strategy to use to recover from unhandled exceptions without causing the projection to fail
   recovery-strategy {
-    # fail - Fail the projection if the handler throws an exception
-    # skip - Skip to the next envelope
-    # retry-and-fail - Retry the handler with the same envelope `retries` before failure
-    # retry-and-skip - Retry the handler with the same evvelope `retries` before moving to the next record
+    # fail - If the first attempt to invoke the handler fails it will immediately give up and fail the stream.
+    # skip - If the first attempt to invoke the handler fails it will immediately give up, discard the element and
+    #        continue with next.
+    # retry-and-fail - If the first attempt to invoke the handler fails it will retry invoking the handler with the
+    #                  same envelope this number of `retries` with the `delay` between each attempt. It will give up
+    #                  and fail the stream if all attempts fail.
+    # retry-and-skip - If the first attempt to invoke the handler fails it will retry invoking the handler with the
+    #                  same envelope this number of `retries` with the `delay` between each attempt. It will give up,
+    #                  discard the element and continue with next if all attempts fail.
     strategy = fail
 
-    # The Number of times to retry handler function
-    # Only applicable to `retry-and-fail` and `retry-and-skip` recovery strategies
+    # The number of times to retry handler function
+    # This is only applicable to `retry-and-fail` and `retry-and-skip` recovery strategies
     retries = 1
 
-    # Delay between retry attempts
+    # The delay between retry attempts.
     # Only applicable to `retry-and-fail` and `retry-and-skip` recovery strategies
     retry-delay = 0 s
```

Comment:
then you can change the defaults here instead of in my https://github.com/akka/akka-projection/pull/178 to avoid a merge conflict

Reply:
:+1: